### PR TITLE
Additional features should now work with the forumlist as well

### PR DESCRIPTION
**Pull request by annando**
Now, finally, the additional features should work as expected.

### CODE REVIEW

File: include/features.php [hunk 1]
```diff
@@ -16,14 +16,10 @@
...
-		if (!$x) {
-			$x = Config::get('feature', $feature, false);
-			if (!$x) {
-				$x = get_feature_default($feature);
-			}
-		}
+	if ($x === false) {
+		$x = get_feature_default($feature);
+		$x = Config::get('feature', $feature, $x);
+		$x = PConfig::get($uid, 'feature', $feature, $x);
```
Comment (MrPetovan) [inline include/features.php]: This always performs at least 3 SQL queries while the previous structure perfomed at most 3 SQL queries, why the change?

Also, line 17 could have been included in the flattening as well, becoming the last call after line 22.

Comment (annando) [reply include/features.php]: The change was a part of a longer debug experience :-) I thought it would look nicer.

Comment (MrPetovan) [reply include/features.php]: It does indeed, at the price performance, and I wondered why you didn’t go all the way.

Comment (annando) [reply include/features.php]: Yeah, it seems that you are right.

Comment (annando) [reply include/features.php]: I cannot include the line 17 in it, since this line overrides the other lines. BTW: I don't think that this is a huge performance, since we are caching the result anyway.

What do you think?

Comment (MrPetovan) [reply include/features.php]: If you still want to cache it without unnecessary calls, you can do it this way:
````php
$x = Config::get('feature_lock', $feature, false);
if ($x === false) {
    $x = PConfig::get($uid, 'feature', $feature, false);
}
if ($x === false) {
    $x = Config::get('feature', $feature, false);
}
if ($x === false) {
    $x = get_feature_default($feature);
}
````
